start.py

```diff
@@ -16,42 +17,66 @@
 
 logger.info('Started Mendix Cloud Foundry Buildpack')
 
-nginx_port = int(os.environ['PORT'])
-runtime_port = nginx_port + 1
-admin_port = runtime_port + 1
+def get_nginx_port():
+    return int(os.environ['PORT'])
+
+
+def get_runtime_port():
+    return get_nginx_port() + 1
+
+
+def get_admin_port():
+    return get_runtime_port() + 1
+
+
+def get_deploy_port():
+    return get_admin_port() + 1
```

Comment:
As the logic is now less close together, it might make sense to use +1, +2 and +3. That's cognitively easier to grok.